__tests__/cli.test.ts

```diff
@@ -238,6 +249,34 @@ describe('CLI', () => {
     expect(await cli.exitCode).toBe(0);
     process.env['TS_NODE_TYPE_CHECK'] = 'false';
   });
+
+  describe('testing a TLS site with self-signed cert', () => {
+    let tlsServer: Server;
+    let cliArgs: Array<string>;
+
+    beforeAll(async () => {
+      tlsServer = await Server.create({ tls: true });
+      cliArgs = [
+        join(FIXTURES_DIR, 'example.journey.ts'),
+        '--params',
+        JSON.stringify({ url: tlsServer.TEST_PAGE }),
+        '--reporter',
+        'json',
+      ];
+    });
+
+    afterAll(async () => await tlsServer.close());
+
+    it('fails by default', async () => {
+      const cli = new CLIMock(cliArgs);
+      expect(await cli.exitCode).toBe(1);
+    });
+
+    it('succeeds succeeds with --ignoreHTTPSErrors', async () => {
+      const cli = new CLIMock([...cliArgs, '--ignoreHTTPSErrors']);
+      expect(await cli.exitCode).toBe(0);
```

Comment:
would look for some events to exist, just for double checking.

Reply:
Done